seatunnel-e2e/seatunnel-connector-v2-e2e/connector-redis-e2e/src/test/java/org/apache/seatunnel/e2e/connector/redis/RedisTestCaseTemplateIT.java

```diff
@@ -343,12 +350,51 @@ public void testScanZsetTypeWriteRedis(TestContainer container)
             disabledReason = "Currently FLINK do not support multiple table read")
     public void testMultipletableRedisSink(TestContainer container)
             throws IOException, InterruptedException {
-        Container.ExecResult execResult =
-                container.executeJob("/fake-to-multipletableredissink.conf");
+        Container.ExecResult execResult = container.executeJob(testMultipletableRedisSinkConf());
         Assertions.assertEquals(0, execResult.getExitCode());
         jedis.select(3);
         Assertions.assertEquals(2, jedis.llen("key_multi_list"));
         jedis.del("key_multi_list");
         jedis.select(0);
     }
+
+    ///// need different redis version test override
```

Comment:
We do need this any more?

Reply:
No need at this stage, so removed redundant code abstractions in next commit